application/helpers/frontend_helper.php

```diff
@@ -2214,3 +2214,29 @@ function cookieConsentLocalization()
         gT('Please be patient until you are forwarded to the final URL.')
     );
 }
+
+/**
+ * Returns an array of URL parameters that can be forwarded
+ *
+ * @return array<string,mixed>
+ */
+function getForwardParameters()
```

Comment:
Hm, this always returns `[]` if it's not GET call?

Reply:
Check now please.